libs/sdk-core/src/breez_services.rs

```diff
@@ -294,6 +294,9 @@ impl BreezServices {
         {
             Some(_) => Err(SendPaymentError::AlreadyPaid),
             None => {
+                self.validate_peers_connected()?;
```

Comment:
Good point about persisted state. Yes, the info could be stale.

Unfortunately the only error we get from GL is "ran out of routes to try" after it keeps trying to pay for about a minute.

On the other hand, GL is managing the peers, so it's trying to connect to one if unconnected. So it could be that calling `send_payment` with no peers will still work, as it could connect to the peer while trying to pay.

Then overall, IMO it's better if I remove this.

Reply:
Done: ef80e5f891c695314c761af9412f93cdaed39111